pdata/internal/metrics.go

```diff
@@ -191,9 +215,7 @@ func (at MetricAggregationTemporality) String() string {
 //
 // Must use NewMetricDataPointFlagsStruct function to create new instances.
 // Important: zero-initialized instance is not valid for use.
-type MetricDataPointFlagsStruct struct {
-	orig *uint32
-}
+type MetricDataPointFlagsStruct = MetricDataPointFlags
```

Comment:
Should not be needed just make the alias `= internal.MetricDataPointFlags`

Reply:
I can do that for the alias, but don't we need `MetricDataPointFlagsStruct` in `metrics.go` still so that the 4 `FlagsStruct` methods can continue to return a `MetricDataPointFlagsStruct`?

Reply:
Or should I change those methods to return `MetricDataPointFlags` instead

Reply:
They can return MetricDataPointFlags since it is an "alias".

Reply:
Done